src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -132,8 +132,10 @@ class SearchControls extends Component {
           </div>
           <div className="columns medium-3">
             <label htmlFor="facilityType">Select Facility Type</label>
-            <div tabIndex="1" className={`facility-dropdown-wrapper ${facilityDropdownActive ? 'active' : ''}`} onClick={this.toggleFacilityDropdown} onBlur={() => {this.setState({ facilityDropdownActive: false });}}>
-              {this.renderSelectOptionWithIcon(currentQuery.facilityType)}
+            <div tabIndex="1" className={`facility-dropdown-wrapper ${facilityDropdownActive ? 'active' : ''}`} onClick={this.toggleFacilityDropdown} onBlur={() => {setTimeout(() => {this.setState({ facilityDropdownActive: false });}, 200);}}>
+              <div className="flex-center">
```

Comment:
Every 2ms or 2 sec? This timeout only affects the facility type dropdown - I did see the zooming sporadically but only if there are no results in the searched region..

Do you have geolocation on or is it the initial request for Washington DC?